randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -117,9 +124,18 @@ Extra - asset_id: 3
   * Layers: default
   * Normal Door to Crateria Kihunter Room/Door to Moat
   > Door to West Ocean
-      Grapple Beam or Space Jump
+      Any of the following:
+          Grapple Beam or Space Jump
+          Speed Booster and Short Charge (Intermediate) and Disabled Entrance Randomizer
+          Gravity Suit and Gravity Jump (Intermediate) and Knowledge (Intermediate)
+          All of the following:
+              # Using a CWJ to get across
+              Knowledge (Intermediate) and Movement (Advanced) and Wall Jumps (Expert)
```

Comment:
Could add HBJ.

Reply:
How? Just via IBJ advanced? should I do a seperate HBJ trick? never did any HBJs so cant judge their difficulty level